Radicle/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM rustlang/rust:nightly
+ENV RUST_LOG=debug
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN apt-get install -y nodejs
+RUN apt-get update && apt-get -y install cmake

```

I think it is best to run `apt-get update` first unless there is a reason not to. Then you can combine the install commands
```suggestion
RUN apt-get update
RUN apt-get -y install cmake nodejs
```